# Environemnt parameters
map_yaml_filename: 'config_FTM_Halle.yaml'
odom_topic: '/ego_racecar/odom'
# odom_topic: '/pf/pose/odom'

# MPC prediction paramter
t: 10
goal_weight_x: 13.5
goal_weight_y: 13.5
goal_weight_v: 5.5
goal_weight_theta: 13.0
ctrl_penalty_a: 0.01
ctrl_penalty_steer: 100

n_ind_search: 5
dt: 0.10
dl: 0.2
varying_dl: false
dl_factor_num: 0.15
dl_factor_denom: 2.5
control_frequency: 10

# Vehicle parameters
wheelbase: 0.3302
max_steer: 24.0 # degrees
max_dsteer: 180.0 # degrees/s
max_speed: 2.9
min_speed: 0
max_accel: 2.5
speed_factor: 0.93